test/main/services/team/TeamProvisioningInboxRelayPolicy.test.ts

```
@@ -0,0 +1,92 @@
+      },
+      {
+        ...baseMessage,
+        messageId: 'nudge',
+        messageKind: 'member_work_sync_nudge' as const,
+        timestamp: '2026-04-24T12:04:00.000Z',
+      },
+      { ...baseMessage, messageId: 'normal-older', timestamp: '2026-04-24T12:01:00.000Z' },
+    ];
+
+    expect(
+      messages.sort(compareOpenCodeInboxRelayMessagesByPriority).map((m) => m.messageId)
```

ESLint

[error] Move this array "sort" operation to a separate statement or replace it with "toSorted".

(sonarjs/no-misleading-array-reverse)